### PR TITLE
Add footnote clarifying extreme weather

### PR DESCRIPTION
**Pull request by ianmurrays**

### CODE REVIEW

File: legislation.tex [hunk 1]
```diff
@@ -67,7 +67,7 @@ \chapter{Cake invoking circumstances}\label{chp:circumstances}
     \emph{COR}.
   \item Using incorrect recipient in sufficiently important email.
   \item Having passed an exam, either final exam or getting the top grade.
-  \item Extreme weather (\emph{cake responsible organization} is Clearhaus A/S).
+  \item Extreme weather (\emph{cake responsible organization} is Clearhaus A/S).\footnote{As defined by DMI as a Category 3 warning in the Aarhus area.}
```
Comment (mt-clearhaus) [inline legislation.tex]: Perhaps `s/warning/Warning/`?

How about `s/3/2/`? :slightly_smiling_face: Aarhus is almost never hit by very extreme weather...

http://www.dmi.dk/vejr/tjenester/varsler/

Comment (ianmurrays) [reply legislation.tex]: Not sure what the rest think of lowering to 2

Comment (kse-clearhaus) [reply legislation.tex]: I approve of category 2 :+1:

**Comment by ianmurrays** [comment]
Will someone render the PDF?

**Review by kse-clearhaus — APPROVED** [review]
:+1:

**Comment by ianmurrays** [comment]
Why you commit in lowercase 😭

**Comment by kse-clearhaus** [comment]
I will be more careful in the future, did not mean to hurt your feelings :hugs:

**Pull request merged**